### PR TITLE
[zh-cn] sync configmap pr-wranglers

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/contribute/participate/pr-wranglers.md [hunk 1]
```diff
@@ -92,6 +86,19 @@ Each day in a week-long shift as PR Wrangler:
...
     这有助于接纳新的贡献者。
 
+<!--
+- Also check for pull requests against the [reference docs generator](https://github.com/kubernetes-sigs/reference-docs) code, and review those (or bring in help).
+- Support the [issue wrangler](/docs/contribute/participate/issue-wrangler/) to
+  triage and tag incoming issues daily.
+  See [Triage and categorize issues](/docs/contribute/review/for-approvers/#triage-and-categorize-issues)
+  for guidelines on how SIG Docs uses metadata.
+-->
+- 同时检查针对[生成文档](https://github.com/kubernetes-sigs/reference-docs)的代码拉取请求，
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/participate/pr-wranglers.md:96]:
```suggestion
- 同时检查针对[参考文档生成器](https://github.com/kubernetes-sigs/reference-docs)的代码拉取请求，
```

File: content/zh-cn/docs/contribute/participate/pr-wranglers.md [hunk 2]
```diff
@@ -92,6 +86,19 @@ Each day in a week-long shift as PR Wrangler:
...
+  并对其进行审查（或寻求帮助）。
+- 支持[问题管理者](/zh-cn/docs/contribute/participate/issue-wrangler/)每日对新问题进行分类和标记。
+  参见[分类和组织问题](/zh-cn/docs/contribute/review/for-approvers/#triage-and-categorize-issues)
+  了解 SIG Docs 如何使用元数据的指南。
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/participate/pr-wranglers.md:100]:
```suggestion
  了解 SIG Docs 如何使用元数据。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**